### PR TITLE
r-rgraphviz: Create new package.

### PR DESCRIPTION
**Pull request by lingnanyuan**
Add package: r-rgraphviz.

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-rgraphviz/package.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RRgraphviz(RPackage):
```
Comment (davydden) [inline var/spack/repos/builtin/packages/r-rgraphviz/package.py]: `Rgraphviz`?

Comment (lingnanyuan) [reply var/spack/repos/builtin/packages/r-rgraphviz/package.py]: The package name is Rgraphviz, and it's r package, so add an "R" before the package name.

Comment (davydden) [reply var/spack/repos/builtin/packages/r-rgraphviz/package.py]: but `Rgraphviz` i think already indicates that it's `R` package of `graphviz`, so one `R` should be enough, isn't it?

Comment (lingnanyuan) [reply var/spack/repos/builtin/packages/r-rgraphviz/package.py]: No, I tried to delete one R, it reported error:  Error: 'module' object has no attribute 'RRgraphviz'. So I think two Rs are needed. Some other R packages starting with r also have two Rs.

Comment (davydden) [reply var/spack/repos/builtin/packages/r-rgraphviz/package.py]: ok, just looked strange

**Pull request merged**